docs/how-to-guides/widgets/legacy-widget-block.md

```diff
@@ -191,3 +191,5 @@ add_action( 'enqueue_block_editor_assets', function() {
 	wp_add_inline_script( 'wp-widgets', 'wp.widgets.registerLegacyWidgetBlock()' );
 } );
 ```
+
+To load the styles and scripts on the editor for the Legacy Widget block, it's recommended to use constructore of the widget registration class or a widget method. It will ensure style and script being loaded on the iFrame as well.
```

Comment:
Thank you for this suggestion @sagarnasit.
I'm not sure if we want to make that an official recommendation. But I guess we could phrase it as:
```
A possible way to ensure the styles and scripts are loaded on the editor for the Legacy Widget block, is to enqueue the assents on the constructor of the widget registration class or a widget method. It will ensure style and script are loaded on the iFrame as well.
```

That way we still provide useful information to developers without sounding like it is an official guideline. What do you think? Thank you in advance for your thoughts/insights.